ci/helpers/aws_helper.go

```diff
@@ -139,6 +139,10 @@ func CleanUpParameterGroups(prefix string, session *session.Session) error {
 }
 
 func CleanUpTestDatabaseInstances(prefix string, awsSession *session.Session) error {
+	if !strings.HasPrefix(prefix, "build-test-") {
+		panic("Trying to clean up databases without the 'build-test-' prefix will fail")
+	}
+
 	rdsSvc := rds.New(awsSession)
```

Comment:
This check should be added to the CleanUpParameterGroups function too.

Reply:
Good catch